apps/web/public/sw.js

```diff
@@ -0,0 +1,411 @@
+/* Service Worker */
```

Comment:
This is the implementation of service worker which intercepts all fetch/network requests coming from browser and acts as a proxy between client and server/cached responses.